public/scripts/app.js

```diff
@@ -0,0 +1,27 @@
+// app.js //
+angular
```

Comment:
It looks like there are a mix of soft and hard tabs. Check out whitespace set-up in the text editor, it may have some formatting tools that are making changes.